GitUI/UserControls/FileStatusList.cs

```diff
@@ -155,11 +155,11 @@ public bool FilterVisible
             }
         }
 
-        public override bool Focused
+        public bool FileListFocused
```

Comment:
`FileListFocused` should not replace `Focused`. It should be added as a separate function and called when needed.
Otherwise, it reverts changes from:
https://github.com/gitextensions/gitextensions/commit/ee3ba236fe9027cff450520829eafd766d42f02c (#3920)

Reply:
@pmiossec & @jbialobr 
I can't see how and where the `Focused` gets used. Is there any way to reproduce the problem?

Reply:
@EbenZhang: Having a bit of play with this, I think that we just need revert the override to pre-@pmiossec's changes:

```
        public override bool Focused
        {
            get
            {
                return FileStatusListView.Focused;
            }
        }
```

and revert all your changes to FormCommit.

Reply:
I think `FileListFocused` is a better name than `Focused`.  
People will be surprised when the Combox is focused but the `Focused` returns `false`.

Reply:
There is a property `ContainsFocus` to tell whether `FileStatusList` control or one of its child controls currently has the input focus
https://msdn.microsoft.com/en-us/library/system.windows.forms.control.containsfocus(v=vs.110).aspx

Reply:
Using the `Focused` as the name is a bad design because developer needs to know implementation details of `Focused` and keep in mind the `Focused` may not return `True` even if the control is `Focused`. He needs to call both `Focused` and `ContainsFocus` to make sure control is focused, which sounds really odd.